Feature partial match

cyang-kth opened the pull request.

ziXet commented:
I'm interested for test this feature. But, there are major conflicts with the master branch.

cyang-kth commented:
@ziXet 

Thanks for your interest in contributing to this repo. We have already implemented this feature but in another private branch/repo. We are working on a new paper on map matching where we have significantly refactored the API and implemented many other new functionalities and algorithms. To protect our research idea, we will not push these changes until the paper is published. So in a short while, we will not make too much change to this repo. Sorry for this convenience. My suggestion is that you could continue to work on your fork and after our update, we would check if it could be merged.
